test/plugins/features/movies/controller.test.js

```diff
@@ -38,35 +49,55 @@ describe('movie controller', () => {
       const movies = await Controller.list(query);
 
       expect(movies.length).to.eql(1);
-      expect(movies.models[0].get('name')).to.eql(payload.name);
-      expect(movies.models[0].get('release_year')).to.eql(payload.release_year);
+      expect(movies.models[0].get('name')).to.eql(testMovie1.name);
+      expect(movies.models[0].get('release_year')).to.eql(testMovie1.release_year);
     });
 
     it('retrieves movies in a range of years', async () => {
       const query = { from_year: 2000, to_year: 2010 };
       const movies = await Controller.list(query);
 
       expect(movies.length).to.eql(1);
-      expect(movies.models[0].get('name')).to.eql(payload.name);
-      expect(movies.models[0].get('release_year')).to.eql(payload.release_year);
+      expect(movies.models[0].get('name')).to.eql(testMovie1.name);
+      expect(movies.models[0].get('release_year')).to.eql(testMovie1.release_year);
     });
 
     it('retrieves a movie based on the title', async () => {
       const query = { title: 'Zodiac' };
       const movies = await Controller.list(query);
 
       expect(movies.length).to.eql(1);
-      expect(movies.models[0].get('name')).to.eql(payload.name);
-      expect(movies.models[0].get('release_year')).to.eql(payload.release_year);
+      expect(movies.models[0].get('name')).to.eql(testMovie1.name);
+      expect(movies.models[0].get('release_year')).to.eql(testMovie1.release_year);
     });
 
     it('retrieves a movie based on the fuzzy title', async () => {
       const query = { title: 'apes' };
       const movies = await Controller.list(query);
 
       expect(movies.length).to.eql(1);
-      expect(movies.models[0].get('name')).to.eql(payload2.name);
-      expect(movies.models[0].get('release_year')).to.eql(payload2.release_year);
+      expect(movies.models[0].get('name')).to.eql(testMovie2.name);
+      expect(movies.models[0].get('release_year')).to.eql(testMovie2.release_year);
+    });
+
+  });
+
+  describe('retrieve movies with locations', () => {
+
+    beforeEach(async () => {
```

Comment:
if you manage to move the `describe` block up in scope, this `beforeEach` would no longer be needed too. However, the outer block should probably have the functionality to `TRUNCATE` + insert the movies/locations/locations_movies resources too.